### PR TITLE
Class remap subscenario

### PR DESCRIPTION
**Pull request by TLESORT**
[To merge after #245 and #246] A utility function to remap classes, this utility function should be applied by the user to make clear that a remapping is executed

### CODE REVIEW

File: continuum/scenarios/scenario_utils.py [hunk 1]
```diff
@@ -4,39 +4,123 @@
...
 
+def _get_remapping_classes_ascending_order(new_classes, current_mapping=None):
+    """
+    Output a vector of classes existing class to get new class labels do:
+     new_label = np.where(new_remapping==old_label)[0][0]
+
+    :param new_classes: list of new classes
+    :param current_mapping: vector with previous remapping
+    """
+
```
Comment (arthurdouillard) [inline continuum/scenarios/scenario_utils.py:19]: Stop putting empty line like this after function doc

File: continuum/scenarios/scenario_utils.py [hunk 2]
```diff
@@ -4,39 +4,123 @@
...
+    else:
+        # remove classes already in the mapping
+        array_new_classes = np.setdiff1d(array_new_classes, current_mapping)
+        if len(array_new_classes) == 0:
+            new_remapping = current_mapping
+        else:
+            new_remapping = np.concatenate([current_mapping, array_new_classes], axis=0)
+
+    return new_remapping
+
```
Comment (arthurdouillard) [inline continuum/scenarios/scenario_utils.py:38]: Two empty lines between functions

File: continuum/scenarios/scenario_utils.py [hunk 3]
```diff
@@ -4,39 +4,123 @@
...
 
 from continuum.datasets import InMemoryDataset, H5Dataset
-from continuum.scenarios import ContinualScenario
+from continuum.scenarios import ContinualScenario, OnlineFellowship
 from continuum.tasks import TaskType
 
 
+def _get_remapping_classes_ascending_order(new_classes, current_mapping=None):
+    """
+    Output a vector of classes existing class to get new class labels do:
```
Comment (arthurdouillard) [inline continuum/scenarios/scenario_utils.py:13]: I don't understand what does that mean

File: continuum/scenarios/scenario_utils.py [hunk 4]
```diff
@@ -4,39 +4,123 @@
...
+    new_vector = np.ones(len(class_vector)) * -1
+    for i, key in enumerate(remapping):
+        indexes = np.where(class_vector == key)[0]
+        new_vector[indexes] = i
+
+    if len(np.where(new_vector == -1)[0]) > 0:
+        raise ValueError("Some indexes have not been set in the remapping.")
+    return new_vector
+
+def remap_class_vector(class_vector, remapping=None):
```
Comment (arthurdouillard) [inline continuum/scenarios/scenario_utils.py:57]: Isn't it the mapping already done in https://github.com/Continvvm/continuum/blob/master/continuum/scenarios/class_incremental.py#L69 ? (much simpler and probably faster), or am i missing something?

File: continuum/scenarios/scenario_utils.py (hunk 1 above)
Comment (TLESORT) [reply continuum/scenarios/scenario_utils.py:19]: C'est pas corrigé automatiquement par l'automatique code formatting, c'est quoi le probleme? ca fait bugger la documentation automatique? Visuellement ca perturbe rien en vrai....

Comment (arthurdouillard) [reply continuum/scenarios/scenario_utils.py:19]: It's a convention, you can check good codes, and there are mostly formatted without that empty line.

Auto formatting is not perfect. Moreover, it's not that hard to remove those lines, and it'll keep the code cleaner, and with a consistent coding style.

Comment (arthurdouillard) [reply continuum/scenarios/scenario_utils.py:19]: As an example: if you add an empty line between each line of code, I'm sure your formatter won't fix that, but that's still a bit ugly don't you think?

Comment (TLESORT) [reply continuum/scenarios/scenario_utils.py]: That do not disturb me anyhow... anyway I think I fixed it in last commit.

**Comment by arthurdouillard** [comment]
Need doc web

**Comment by TLESORT** [comment]
> Need doc web

done

**Pull request merged**